lib/router/bindDefaultHandlers.js

```diff
@@ -77,7 +81,8 @@ module.exports = function(sails) {
       // Log a message and try to use `res.send` to respond.
       try {
         sails.log.verbose('A request did not match any routes, and no `res.notFound` handler is configured.');
-        res.send(404);
+        res.status(404);
+        res.send('Not found');
```

Comment:
I'm not sure what's going on, but when I tried that, I got `Cannot call method 'send' of undefined`, the response from `res.status` was undefined.